docs/sources/reference/commandline/cli.md

```diff
@@ -1125,7 +1125,8 @@ and Docker images will report:
 
 The `--since` and `--until` parameters can be Unix timestamps, RFC3339
 dates or Go duration strings (e.g. `10m`, `1h30m`) computed relative to
-client machine’s time.
+client machine’s time. If `--since` is not provided, only new/live events be
+shown
```

Comment:
Same here;

    If the `--since` option is not provided, only new/live events will be show.